### ISSUE TITLE
Craft 5 + Formie + Feed Me = Feed Me No Workie

### ISSUE THREAD

**Issue by chadcrowell** [verbb/formie#1774]
### Describe the bug

I have a new Craft 5 installation I've been working on. I've had Formie installed on it for a few weeks and it has been working fine. P&T just released Feed Me for Craft 5. I installed it and clicked the New Feed button, and am getting this error:

```
yii\base\ErrorException: Type of verbb\formie\integrations\feedme\elements\Submission::$element must not be defined (as in class craft\feedme\base\Element) in /var/www/html/vendor/verbb/formie/src/integrations/feedme/elements/Submission.php:0
Stack trace:
#0 [internal function]: yii\base\ErrorHandler->handleFatalError()
#1 {main}
```

Could be a Feed Me issue since it is the new piece of this puzzle, but since the error references Formie, I thought I'd check here. Thanks.

### Steps to reproduce

1. Install Craft 5
2. Install Formie
3. Install Feed Me
4. Click New Feed
5. Get error

### Form settings

No forms created yet.

### Craft CMS version

Craft CMS  5.0.0-beta.10

### Plugin version

3.0.0-beta.3

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.3"`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.4](https://github.com/verbb/formie/releases/tag/3.0.0-beta.4)
